src/reducers/biblioReducer.js

```diff
@@ -980,7 +980,7 @@ export default function(state = initialState, action) {
         referenceCurie: action.payload,
         entityAdd: setRefCurieDefaultEntityAdd,
 	    tetPageSize: defaultTetPageSize,
-        curieToNameTaxon: [],
+        curieToNameTaxon: {'NCBITaxon:7955': 'Danio rerio', 'NCBITaxon:8364': 'Xenopus tropicalis'},
```

Comment:
Why do we need these values ?  I thought they should come from components/biblio/topic_entity_tag/TaxonUtils.js getCurieToNameTaxon

Reply:
I need to remove this... this was for testing.

Reply:
cool cool

Reply:
This has been removed...